kafka-ui-react-app/src/lib/yupExtended.ts

```diff
@@ -31,15 +32,40 @@ export const isValidJsonObject = (value?: string) => {
   return false;
 };
 
-const isJsonObject = () => {
+const isJsonObject = (message?: string) => {
   return yup.string().test(
     'isJsonObject',
     // eslint-disable-next-line no-template-curly-in-string
-    '${path} is not JSON object',
+    message || '${path} is not JSON object',
     isValidJsonObject
   );
 };
 
+const isValidEnum = (value?: string) => {
+  try {
+    if (!value) return false;
+    const trimmedValue = value.trim();
+    if (
+      trimmedValue.indexOf('e') === 0 &&
```

Comment:
i think this check `e`  could make invalid values true like `eWrong = {}`

Reply:
i replace 'e'  with 'enum', also added test cases